packages/suite/src/components/wallet/CoinmarketTransactionId/index.tsx

```diff
@@ -10,27 +13,58 @@ interface Props {
 
 const Wrapper = styled.div`
     display: flex;
+    margin-top: 10px;
+    padding: 0 10px;
+    align-items: center;
+    justify-content: space-between;
     font-size: ${variables.FONT_SIZE.TINY};
     color: ${props => props.theme.TYPE_LIGHT_GREY};
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
 `;
 
+const Left = styled.div`
+    display: flex;
+`;
+
+const Right = styled.div``;
+
 const Label = styled.div`
     text-transform: uppercase;
     font-size: ${variables.FONT_SIZE.TINY};
     padding-right: 3px;
     white-space: nowrap;
 `;
 
-const Value = styled.div``;
+const Value = styled.div`
+    max-width: 220px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+`;
 
 const TransactionId = ({ transactionId, className }: Props) => {
+    const htmlElement = createRef<HTMLDivElement>();
+    const { addNotification } = useActions({ addNotification: notificationActions.addToast });
+    const copy = () => {
+        const result = copyToClipboard(transactionId, htmlElement.current);
```

Comment:
I think `copyToClipboard(transactionId, null)` would be the same as you don't really need or set htmlElement.current to anything.